claim-db-worker/app/web/layout.tsx

```
@@ -0,0 +1,342 @@
+
+  return (
+    <>
+      <div className="w-full max-w-7xl mx-auto px-2 sm:px-4 py-8 sm:py-12 font-barlow">
+        <div className="flex-1 min-h-[calc(100vh-280px)]">
+          <div className="w-full h-full">
+            <div className="flex flex-col md:flex-row-reverse justify-between">
+              <div className="flex lg:hidden flex-row gap-1 mb-4 lg:mb-0 lg:justify-end lg:items-center">
+                <div
+                  className="flex-1 px-4 py-2 bg-teal-600 rounded-md flex justify-center items-center cursor-pointer hover:bg-teal-700 transition-colors"
+                  onClick={handleClaimDatabase}
+                >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)